fiaas_deploy_daemon/specs/v3/factory.py

```diff
@@ -220,6 +220,7 @@ def _labels_annotations_spec(labels_annotations_lookup, overrides):
             "service_account": dict(labels_annotations_lookup["service_account"]),
             "pod": dict(labels_annotations_lookup["pod"]),
             "status": {},
+            "pod_disruption_budget": {},
```

Comment:
I missed this on my first review; is there any reason not to support setting labels or annotations on pod disruption budget from the FIAAS config (via labels_annotations_lookup here)? While I can't think of a particular reason it is explicitly needed, it is supported for the other managed resources (except for the (application)status, which is a bit different anyway)

Reply:
Just an oversight. I added it now.